diff --git a/deps/sqlite3.gyp b/deps/sqlite3.gyp
index 6e377a849006913fba3afdda10647d03328dc6e5..3b2a811ea77e6cedf6aa0777324d09c07b801139 100755
--- a/deps/sqlite3.gyp
+++ b/deps/sqlite3.gyp
@@ -44,7 +44,7 @@
               '<(SHARED_INTERMEDIATE_DIR)/sqlite3/>(openssl_root)/libcrypto.lib',
               '<(SHARED_INTERMEDIATE_DIR)/sqlite3/signal-tokenizer/>(rust_arch)-pc-windows-msvc/signal_tokenizer.lib',
             ],
-            'destination': '<(PRODUCT_DIR)',
+            'destination': '<(module_root_dir)',
           }],
         }],
       ],
@@ -78,17 +78,17 @@
           ],
           'link_settings': {
             'libraries': [
-              '-llibcrypto.lib',
-              '-llibssl.lib',
+              '-l<(module_root_dir)/libcrypto.lib',
+              '-l<(module_root_dir)/libssl.lib',
               '-lws2_32.lib',
               '-lcrypt32.lib',
               '-lbcrypt.lib',
               '-luserenv.lib',
               '-lntdll.lib',
-              '-lsignal_tokenizer.lib',
+              '-l<(module_root_dir)/signal_tokenizer.lib',
             ],
             'library_dirs': [
-              '<(PRODUCT_DIR)',
+              '<(module_root_dir)',
             ]
           }
         },
diff --git a/lib/database.js b/lib/database.js
index 5d6f7a1943214d5147573a9d365e200564c70b78..0b4a341f78f3749ffe9190afa3da72923bef5e7e 100644
--- a/lib/database.js
+++ b/lib/database.js
@@ -43,11 +43,14 @@ function Database(filenameGiven, options) {
 	if (nativeBindingPath != null && typeof nativeBindingPath !== 'string') throw new TypeError('Expected the "nativeBinding" option to be a string');
 
 	// Load the native addon
+	var requireFunc = typeof __webpack_require__ === 'function'
+      ? __non_webpack_require__
+      : require;
 	let addon;
 	if (nativeBindingPath == null) {
-		addon = DEFAULT_ADDON || (DEFAULT_ADDON = require('bindings')('better_sqlite3.node'));
+		addon = DEFAULT_ADDON || (DEFAULT_ADDON = requireFunc('bindings')('better_sqlite3.node'));
 	} else {
-		addon = require(path.resolve(nativeBindingPath).replace(/(\.node)?$/, '.node'));
+		addon = requireFunc(path.resolve(nativeBindingPath).replace(/(\.node)?$/, '.node'));
 	}
 	if (!addon.isInitialized) {
 		addon.setErrorConstructor(SqliteError);
diff --git a/package.json b/package.json
index ba77120906d6a37331458e11bf1eba4c4a3a5ece..b7615b2cb91c286e470ae42b75b4db9b5747d52a 100644
--- a/package.json
+++ b/package.json
@@ -35,8 +35,8 @@
   },
   "scripts": {
     "install": "npm run download && npm run build-release",
-    "build-release": "node-gyp rebuild --release",
-    "build-debug": "node-gyp rebuild --debug",
+    "build-release": "node-gyp rebuild --release --verbose",
+    "build-debug": "node-gyp rebuild --debug --verbose",
     "rebuild-release": "npm run lzz && npm run build-release",
     "rebuild-debug": "npm run lzz && npm run build-debug",
     "test": "mocha --exit --slow=75 --timeout=5000",
