diff --git a/node_modules/@types/backbone/index.d.ts b/node_modules/@types/backbone/index.d.ts
index da53a62..4db8b56 100644
--- a/node_modules/@types/backbone/index.d.ts
+++ b/node_modules/@types/backbone/index.d.ts
@@ -225,7 +225,7 @@ declare namespace Backbone {
         * That works only if you set it in the constructor or the initialize method.
         **/
         defaults(): ObjectHash;
-        id: any;
+        id: string;
         idAttribute: string;
         validationError: any;

